packages/admin-next/dashboard/src/routes/discounts/details/components/discounts-configurations-section/discount-configurations-section.tsx

```diff
@@ -0,0 +1,102 @@
+import { Discount } from "@medusajs/medusa"
+import { Container, Heading, Text } from "@medusajs/ui"
+import { useTranslation } from "react-i18next"
+
+import { ActionMenu } from "../../../../../components/common/action-menu"
+import { PencilSquare } from "@medusajs/icons"
+import { parse } from "iso8601-duration"
+import { useMemo } from "react"
+
+type DiscountConfigurationsSection = {
+  discount: Discount
+}
+
+function formatTime(dateTime?: string) {
+  if (!dateTime) {
+    return
+  }
+  const date = new Date(dateTime)
+  const a = date.toDateString().split(" ").slice(1).join(" ")
+  return `${a} at ${date.getHours()}:${("0" + date.getMinutes()).slice(-2)}`
+}
```

Comment:
I would just use:

```tsx
format(new Date(dateTime), "dd MMM, yyyy, HH:mm:ss")
```
instead, since with this `at` is not being translated, and the above aligns with how dates are displayed across the other pages.